lib/core/Axios.js

```diff
@@ -32,7 +32,7 @@ Axios.prototype.request = function request (config) {
     }, arguments[1])
   }
 
-  config = utils.merge(defaults, this.defaults, { method: 'get' }, config)
+  config = utils.merge({}, defaults, this.defaults, { method: 'get' }, config)
```

Comment:
I guess the tests related to this were supposed to fail. It might have been some preparation for `Default instances / Config merge:
` in #1333

Reply:
I think this merge is fine and shouldn't change any config behavior.
The previous `utils.merge` always constructed a new object, so the `{}` is just for "normalizing" the behavior.